client/src/styles/_button.scss

```diff
@@ -140,3 +140,112 @@
 .waou-survey-button:focus {
   opacity: 0.7;
 }
+
+// Copied from justfix-website
+
+@mixin button-variant($color: $justfix-black) {
+  background-color: $color;
+
+  // mobile-eyebrow:
```

Comment:
yes, definitely - sorry forgot to mention, but I was planning to do that afterwards since styles have still been changing in figma. Some things are following the new system, but not completely. I still think it's worth adding them in like you suggested, and can just adjust as needed after.